fluent/toInterfaceValue_test.go

```diff
@@ -0,0 +1,51 @@
+package fluent_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime/fluent"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+	"github.com/ipld/go-ipld-prime/node/basicnode"
+)
+
+var roundTripTestCases = []struct {
+	desc  string
+	value interface{}
+}{
+	{desc: "Number", value: 100},
+	{desc: "String", value: "hi"},
+	{desc: "Bool", value: "hi"},
```

Comment:
did you mean to make this a boolean? :)

if you use `go test -coverprofile=cover.out && go tool cover -html=cover.out`, you'll be able to switch to your file and see what lines aren't being tested at all.

Reply:
nice! I didn't know that